configureWorkspace/configure.ts

```diff
@@ -369,14 +369,14 @@ async function configureCore(actionContext: IActionContext, options: ConfigureAp
         // Scope serviceNameAndPathRelativeToRoot only to this block of code
         let serviceNameAndPathRelativeToRoot: string;
         if (platformType.toLowerCase().includes('.net')) {
-            let csProjFilePath = await findCSProjFile(rootFolderPath);
-            serviceNameAndPathRelativeToRoot = csProjFilePath.slice(0, -'.csproj'.length);
-            let csProjFileContents = (await fse.readFile(path.join(rootFolderPath, csProjFilePath))).toString();
+            let projFilePath = await findCSProjOrFSProjFile(rootFolderPath);
+            serviceNameAndPathRelativeToRoot = projFilePath;
```

Comment:
What's the impact of *not* trimming the file extension as the previous version did?

Reply:
Ah, ok, I see, the file extension gets used later in the .NET Core Dockerfile `genDockerFile()` generator function.  Since the generator signature is supposed to be generic across platforms, I'm a little leery of making `serviceNameAndPathRelativeToRoot` semantically different *only* for .NET Core.

I wonder if it'd be better to use the `PackageInfo` object to pass along something like the `packageType`, which is already being collected.  That can then be used by the generator function and keep the existing semantics of the `serviceNameAndPathRelativeToRoot`.  (It looks like the .NET Core generator is already using the `PackageInfo.version` to hold the target .NET Framework, so it'd be a similar idea.  Maybe the `artifactName` would be a good fit?)

Reply:
I went with the `artifactName`.